src/assets/levels.json

```diff
@@ -1,11 +1,35 @@
 [
   {
     "type": "money",
+    "goal": 10010
```

Comment:
I think there are too many at this time. Not that it's a problem to have many levels, but perhaps make them a bit more substantial. Say 10250, 10500, 11000, 12000, 15000, 20000?